hdr.go

```diff
@@ -574,3 +575,49 @@ func bitLen(x int64) (n int64) {
 	}
 	return
 }
+
+// CumulativeDistribution returns an ordered list of brackets of the
+// distribution of recorded values.
+func (h *Histogram) CumulativeDistributionWithTicks(ticksPerHalfDistance int32) []Bracket {
+	var result []Bracket
+
+	i := h.pIterator(ticksPerHalfDistance)
+	for i.next() {
+		result = append(result, Bracket{
+			Quantile: i.percentile,
+			Count:    i.countToIdx,
+			ValueAt:  int64(i.highestEquivalentValue),
+		})
+	}
+
+	return result
+}
+
+// Output the percentiles distribution in a text format
+func (h *Histogram) PercentilesPrint(writer io.Writer, ticksPerHalfDistance int32, valueScale float64) (outputWriter io.Writer, err error) {
+	outputWriter = writer
+	dist := h.CumulativeDistributionWithTicks(ticksPerHalfDistance)
+	_, err = outputWriter.Write([]byte(" Value\tPercentile\tTotalCount\t1/(1-Percentile)\n\n"))
+	if err != nil {
+		return
+	}
+	for _, slice := range dist {
+		percentile := slice.Quantile / 100.0
+		inverted_percentile := 1.0 / (1.0 - percentile)
```

Comment:
If percentile is equal to 1.0, 
java version displays "inf"
python version displays an empty string
go display "+Inf"

Maybe good to start converging.

Reply:
solved the last commit